deploy/helm/flightctl/templates/flightctl-cli-artifacts-deployment.yaml

```diff
@@ -0,0 +1,33 @@
+{{ if .Values.api.enabled }}
```

Comment:
It's being used by the UI only, right? Should we then make deployment depending on UI being enabled instead of API?

Reply:
fixed

Reply:
This container serves both standalone and ACM installations, so it should run independently of UI